styles/prosilver/template/pastebin_body.html

```diff
@@ -88,5 +66,27 @@
 </div>
 
 
+<!-- IF S_AUTH_VIEW -->
+<div class="mini-panel navbar">
+	<span class="corners-top"><span></span></span>
```

Comment:
Das Problem ist:
- diese <span>-Kontrukt wurde für die Runden-Ecken verwendet. Das wird jetzt in phpBB 3.1 rein per CSS gelöst.
- daher ist die CSS-Regel für dieses <span> nicht mehr existent. 
- somit ist dieser Code nutzlos, da er nichts ausrichtet.